concordium-consensus/src/Concordium/ImportExport.hs

```diff
@@ -114,11 +118,106 @@ placeholderSectionHeader = SectionHeader 0 0 P1 (BlockHash minBound) 0 0 0 0
 sectionHeaderLength :: Word64
 sectionHeaderLength = fromIntegral $ BS.length $ encode placeholderSectionHeader
 
--- |Write a section to the file handle.  It must be possible to write and seek the handle.
--- The section is written at the current seek location of the handle, and afterwards the seek
--- location is at the end of the the written section.
-writeSection ::
-    (MonadIO m) =>
+-- |Open a file handle for writing a chunk.
```

Comment:
The documentation is not adequate for what the function does. In particular this deletes an existing file.